packages/components/tsup.config.ts

```diff
@@ -1,7 +1,7 @@
 import { defineConfig } from 'tsup';
 
 export default defineConfig({
-  entryPoints: ['src/index.ts'],
+  entryPoints: ['src/index.ts', 'src/components/Icons/index.ts'],
```

Comment:
Proposal: what if we started using lowercase naming for folders? I think it adheres more to the community standard. Folders with a capital first letter, if used at all, are usually for folder containing a single React component and its tests or closely related files.

Reply:
Done!

Reply:
Done